dandiapi/api/views/info.py

```diff
@@ -0,0 +1,18 @@
+from django.conf import settings
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+
+schema_url = (
+    'https://raw.githubusercontent.com/dandi/schema/master/'
+    f'releases/{settings.DANDI_SCHEMA_VERSION}/dandiset.json'
+)
+
+
+@api_view()
+def info_view(self):
+    return Response(
```

Comment:
Tragically this will not self-document on the swagger page, `/api/info/` will just show some generic 200 response placeholder.

The solution would involve defining a `Serializer` that has `schemaVersion` and `schemaURL` defined as fields, then invoking it like this:
```
from drf_yasg.utils import no_body, swagger_auto_schema
...
    @swagger_auto_schema(
        request_body=no_body,
        responses={
            200: ApiInfoSerializer(),
        },
    )
    ```
    
You should probably also use that serializer when generating the response, but this endpoint is so simple it hardly matters.